assets/_sass/base/_variables.scss

```diff
@@ -1,14 +1,13 @@
 // 18F Fonts
 $raleway: Raleway,'Helvetica Neue',Helvetica,Arial,sans-serif;
 $open-sans: 'Open Sans','Helvetica Neue',Helvetica,Arial,sans-serif;
-$brand: '18F Nimbus','Helvetica Neue',Helvetica,sans-serif;
-$nimbus: 'Nimbus','Helvetica Neue',Helvetica,sans-serif;
+$brand: 'Nimbus','Helvetica Neue',Helvetica,sans-serif;
```

Comment:
No the extra files were just there cuz I forgot to remove them the first time I did this. :( see the title on #821 for more context.